Source/Serilog.Exceptions/Core/ExceptionEnricher.cs

```diff
@@ -20,22 +22,38 @@ public sealed class ExceptionEnricher : ILogEventEnricher
             new ReflectionTypeLoadExceptionDestructurer()
         };
 
+        public static readonly IExceptionPropertyFilter FilterIgnoringStackTraceAndTargetId =
+
+#if NET45
```

Comment:
Perhaps we should move these common filters to their own `ExceptionFilter` static class. Perhaps we should do the same with the destructurers.

Reply:
It would contain just one property so I would keep it the way it is. Additionally, moving destructurers property would be a minor backwards-compatibility break.